src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml

```diff
@@ -21,6 +21,7 @@ imports:
     - { resource: "@SyliusUserBundle/Resources/config/app/config.yml" }
     - { resource: "@SyliusVariationBundle/Resources/config/app/config.yml" }
 
+    - { resource: "parameters.yml" }
```

Comment:
These parameters should not override `app/config/parameters.yml`, so we should load core bundle config before app them.